src/main/resources/lib/samples/sample.jelly

```diff
@@ -44,7 +44,12 @@ THE SOFTWARE.
     <st:adjunct includes="io.jenkins.plugins.designlibrary.styles" />
     <st:adjunct includes="io.jenkins.plugins.designlibrary.sample" />
     <st:adjunct includes="io.jenkins.plugins.designlibrary.preCode" />
-    <st:adjunct includes="io.jenkins.plugins.designlibrary.prism" />
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-core.min.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-markup.min.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-clike.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-css.js"/>
+    <script type="text/javascript" src="${resURL}/plugin/prism-api/js/prism-java.min.js"/>
```

Comment:
I hit timing issues with autoloader where languages weren't loaded when our JavaScript runs, I was able to workaround it by explicitly loading languages and then moving our JS to the callback after it was completed but  some other weird things broke

https://github.com/jenkinsci/prism-api-plugin/blob/a890836315b4079c880a40ba6b1906b7e58cd03d/src/main/resources/io/jenkins/plugins/prism.jelly#L15-L16

May re-visit this but it might be okay as is
Not using the provided jelly file also meant I didn't have to load all the plugins that were forced on me as well...

Reply:
Before packaging prism as API plugin I also did load only a couple of languages. But during the migration using the auto-detection worked quite well so I thought it would be more convenient to have it this way. Does the timing problems also occur in the warnings plugin views?

Reply:
don't think so, part of the problem is that we're remote loading files at runtime as we couldn't get jelly to output jelly without being escaped, so we run some javascript to load it, then we need to use the prism API to convert that to the prism DOM

Reply:
I managed to get this working by calling `Prism.highlightElement(element)` as a backstop and it worked fine.